arviz/stats/diagnostics.py

```diff
@@ -732,8 +732,8 @@ def _ess_sd(ary, relative=False):
     ary = np.asarray(ary)
     if _not_valid(ary, shape_kwargs=dict(min_draws=4, min_chains=1)):
         return np.nan
-    ary = _split_chains(ary)
-    return min(_ess(ary, relative=relative), _ess(ary**2, relative=relative))
+    ary = abs(ary - ary.mean())
```

Comment:
Based on benchmarks, posterior just switched to this form (see https://github.com/stan-dev/posterior/pull/266):
```suggestion
    ary = (ary - ary.mean())**2
```

Reply:
Thanks, I will add this. Need to go through loo to make sure we have the latest changes

Reply:
By fixing the tests I realized that change should have been done to mcse_sd not to ess_sd. With the changes I just pushed we are back to 100% match with posterior